### PR TITLE
updated italian translations

### PR DESCRIPTION
**Pull request by nzambello**

### CODE REVIEW

**Comment by tisto** [comment]
@nzambello could you pls rebase with master and fix the merge conflict? I don't get why the other languages are affected. The Travis job should fail if we have outdated i18n.

**Comment by nzambello** [comment]
@tisto merged master into this.
For the Travis job, I'm not sure about `code-analysis:i18n` in .travis.yml because it passes on master and in this branch doing `yarn code-analysis:i18n && echo 'test'` printing 'test' in both cases.

**Comment by tisto** [comment]
@nzambello ok, now I get it. You added new lines to blocks.js. Will check the Travis i18n check.

File: src/config/Blocks.jsx [hunk 1]
```diff
@@ -79,6 +79,14 @@ defineMessages({
     id: 'html',
     defaultMessage: 'HTML',
   },
+  leadimage: {
+    id: 'leadimage',
+    defaultMessage: 'Lead Image Field',
+  },
+  listing: {
+    id: 'listing',
+    defaultMessage: 'Elenco',
```
Comment (sneridagh) [inline src/config/Blocks.jsx]: @nzambello ups, I think this slipped in, right?

Comment (nzambello) [reply src/config/Blocks.jsx]: @sneridagh whoopsie sorry it was a mistake.

**Comment by sneridagh** [comment]
Merging

**Pull request merged**